src/components/library-item/library-item.css

```diff
@@ -48,11 +49,30 @@
     width: 100%;
 }
 
+.library-item-inset-image-container {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-top: -2.5rem;
+    margin-left: 1rem;
+    align-self: flex-start;
+    background-color: rgb(35, 188, 141);
+    width: 3.5rem;
+    height: 3.5rem;
+    z-index: 10;
```

Comment:
We've been trying avoid z-indices because they can complicate interactions with other parts of the UI. In this case, you don't need a z-index because the inset image comes after the featured image, but because the featured image is `position: relative` it still displays over. So you can replace this z-index with `position: relative` to get things to layer correctly without using z-index. The reason is that element ordering without z-index goes:
1. All non-positioned elements, in order of appearance. 
2. All positioned elements, in order of appearance. 
[src](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/Stacking_without_z-index)

Reply:
Thanks!